.github/workflows/ossf-scorecard.yml

```
@@ -0,0 +1,53 @@
+name: Scorecard analysis workflow
+on:
+  push:
+    # Only the default branch is supported.
+    branches:
+    - main
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)